diesel_cli/README.md

```diff
@@ -4,6 +4,15 @@ Diesel CLI
 Diesel CLI is a tool that aids in managing your database schema. Migrations are
 bi-directional changes to your database that get applied sequentially.
 
+Installation
+---------------
+
+The diesel cli requires `mysql` to be installed on your system. If you have
+homebrew installed, you can run `which mysql || brew install mysql` to install
+`mysql` if necessary.
```

Comment:
I'm very skeptical of suggesting to use brew as not everyone is using macOS, and even there, there are other ways to install it (even though I use brew myself and would recommend it).